spec/factories/factories.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :card do
+    original_text "Mom"
+    translated_text  "Мама"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.